modules/@apostrophecms/asset/lib/webpack/media-to-container-queries-loader.js

```diff
@@ -0,0 +1,94 @@
+const postcss = require('postcss');
+
+module.exports = function (source) {
+  const schema = {
+    title: 'Media to Container Queries Loader options',
+    type: 'object',
+    properties: {
+      debug: {
+        type: 'boolean'
+      },
+      transform: {
+        anyOf: [
+          { type: 'null' },
+          { instanceof: 'Function' }
+        ]
+      }
+    }
+  };
+  const options = this.getOptions(schema);
+
+  const mediaQueryRegex = /@media[^{]*{([\s\S]*?})\s*(\\n)*}/g;
```

Comment:
```suggestion
  const mediaQueryRegex = /@media[^{]*{[\s\S]*}\s*(\\n)*}/g;
```

no need for parenthesis and question mark (`*` does the `?`'s "job"), I think

Reply:
tried and the parentheses are needed, otherwise it will select extra content